R/table.R

```diff
@@ -172,6 +172,11 @@
 # Modify columns of a dataframe to a corresponding value in Synapse based on the Synapse type of the
 # column and save the CSV. 
 .saveToCsvWithSchema <- function(schema, values, file) {
-  df <- .convertToSynapseTypeFromSchema(values, schema$columns_to_store)
-  .saveToCsv(df, file)
+  cols <- schema$columns_to_store
+  if (is.vector(cols)) {
```

Comment:
schema coming back from synGet has empty columns_to_store. PythonEmbedInR converted it to an object with name PythonInR_List. 
```
> table <- synGet("syn20186445")
> class(table$columns_to_store)
[1] "PythonInR_List"   "PythonInR_Object" "R6"
```
A non-empty list of columns converted to an R `list`:
```
> pyCols <- pyGet("[Column(name = 'b', columnType = 'BOOLEAN'), Column(name = 'a', columnType = 'STRING', maximumSize = 1)]")
> pyCols
[[1]]
{
  "columnType": "BOOLEAN",
  "concreteType": "org.sagebionetworks.repo.model.table.ColumnModel",
  "name": "b"
}

[[2]]
{
  "columnType": "STRING",
  "concreteType": "org.sagebionetworks.repo.model.table.ColumnModel",
  "maximumSize": 1,
  "name": "a"
}

> class(pyCols)
[1] "list"
```